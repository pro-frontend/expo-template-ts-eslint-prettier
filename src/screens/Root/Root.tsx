import { StatusBar } from 'expo-status-bar';
import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import styled from 'styled-components/native';

const Root = () => {
  return (
    <View style={styles.container}>
      <Text>Open up App.tsx to start working on your app!</Text>
      <Title>This is Entry Point.</Title>
      <StatusBar />
    </View>
  );
};

const Title = styled.Text`
  color: #000;
  text-align: center;
  font-size: 16px;
`;

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});

export { Root };
