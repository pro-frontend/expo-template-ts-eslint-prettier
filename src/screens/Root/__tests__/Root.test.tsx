import React from 'react';
import renderer from 'react-test-renderer';
import { Root } from '../Root';

describe('<Root />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<Root />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
