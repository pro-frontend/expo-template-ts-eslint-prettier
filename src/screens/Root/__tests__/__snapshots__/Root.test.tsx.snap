// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Root /> has 1 child 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Open up App.tsx to start working on your app!
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "#000",
          "fontSize": 16,
          "textAlign": "center",
        },
      ]
    }
  >
    This is Entry Point.
  </Text>
</View>
`;
